index.md

```diff
@@ -38,7 +38,7 @@ Join #crio on [Kubernetes Slack](https://slack.k8s.io/)
 To install on the following operating systems, set the environment variable `$OS` as the appropriate field in the following table:
 
 | Operating system | $OS               |
-| ---------------- | ----------------- |
+|------------------|-------------------|
```

Comment:
Making this change caused the table to highlight every other row.  That's fine, but the backticks also do that too, and you end up with a double highlight, that IMO looks a bit wonky.  I'd vote to use one of these changes, the backticks of the OS names, or this change, but not both.

Reply:
When I opened it with ./script/server, the formatting was the same as before after this change, so I was not aware of this issue with highlighting every other row. I can return it back, as I agree, double highlighting would look strange, I am just confused why I can't see this when opening it locally.